var/spack/repos/builtin/packages/perl-perl6-slurp/package.py

```diff
@@ -0,0 +1,36 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PerlPerl6Slurp(PerlPackage):
+    """Perl6::Slurp - Implements the Perl 6 'slurp' built-in"""
+
+    homepage = "http://search.cpan.org/~dconway/Perl6-Slurp-0.051005/lib/Perl6/Slurp.pm"
+    url      = "http://search.cpan.org/CPAN/authors/id/D/DC/DCONWAY/Perl6-Slurp-0.051005.tar.gz"
+
+    version('0.051005', '6095c8df495c7983f36996ed78c5ead7')
+
+    conflicts('perl@:5.8.0')
```

Comment:
From [the homepage](https://metacpan.org/pod/release/DCONWAY/Perl6-Slurp-0.051005/lib/Perl6/Slurp.pm#DEPENDENCIES), it says:

> Requires: Perl 5.8.0

It sounds like this is actually a backport of a feature added in Perl 6.

Reply:
Yes, it's a backport from perl 6.  The conflict statement requires it to be built with perl 5.8.0 or higher (which shouldn't be an issue since the oldest in spack is 5.16.3), but I added it just in case someone puts in an older version at some point

Reply:
Spack's version ranges are inclusive, so this excludes 5.8.0 too. I would just depend on 5.8.0 specifically.

Reply:
@baberlevi Can you replace this line with:
```python
depends_on('perl@5.8:5.999', type=('build', 'run'))
```
I think that's what they're looking for.